src/duckling/migration/editor-migrations.ts

```diff
@@ -0,0 +1,6 @@
+import {MapMigration} from './map-migration';
+import {MigrationTools} from './migration-tools';
+import {Attribute} from './map-format';
+
+export let editorMigrations : MapMigration[] = [
```

Comment:
Instead of re-using MapMigration here you should create a new interface. Using this forces you to put the migration function on the options object. The options object was intended to be passed to the migration and controls how it behaves, its not supposed to include the migration itself. You should create a new interface for defining Editor migrations.